app/fix-build-plan/Main.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
Wouldn't harm adding some documentation about what `fix-build-plan` does and what the shape/schema expected of the source file is.

It can be explained in the code, but the procedure for using custom versions of packages deserves an explanation in some README for the people interested in using it.